internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -44,10 +43,18 @@ import (
 
 //go:generate mockgen -destination=../../../mocks/mock_quick_start.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/quickstart Flow
 
-const quickstartTemplate = `
-Now you can connect to your Atlas cluster with: mongosh -u %s -p %s %s
+const quickstartTemplateMongoshDetected = `
+MongoDB Shell detected. Connecting to your Atlas cluster:
+$ mongosh -u %s -p %s %s
+`
 
+const quickstartTemplateMongoshNotDetected = `
+MongoDB Shell not detected. To install, open www.mongodb.com/try/download/shell
+[MongoDB Shell (mongosh) is an interactive command line interface to query, update and manage data in the MongoDB database.]
+Once mongosh is installed, you can connect to your Atlas Cluster0 database with:
```

Comment:
@dianchenghu How doy ou feel about a line break here, after the message about mongosh?

I find it hard to read the block of text when it's all together in the example mov.

Reply:
Great point. Yes, it will be better if we add a line break after the install URL.

Other parts looks good!